### PR TITLE
Remove deprcated, update to Pyramid 1.6b3

### PR DESCRIPTION
**Pull request by sbrunner**
The OGC proxy will not be able to access to private networks

### CODE REVIEW

**Comment by sbrunner** [comment]
Ready to be reviewed :-)

File: c2cgeoportal/__init__.py [hunk 1]
```diff
@@ -426,30 +433,59 @@ def default_user_validator(request, username, password):
...
+            ip = IP(gethostbyname(parts.netloc))
+        except gaierror as e:
+            log.info("Unable to get host name for %s: %s" % (url, e))
+            return False
+        for net in self.private_networks:
+            if ip in net:
+                return False
+        return True
+
+    def phash(self):
```
Comment (ochriste) [inline c2cgeoportal/__init__.py]: is this used somewhere ?

Comment (sbrunner) [inline c2cgeoportal/__init__.py]: Yes by Pyramid ...

Comment (sbrunner) [inline c2cgeoportal/__init__.py]: See: http://docs.pylonsproject.org/projects/pyramid/en/1.6-branch/narr/hooks.html#view-and-route-predicates

**Comment by ochriste** [comment]
I cant say I really understand the change in the cache management, so I will trust you on that.
looks ok otherwise
merge after fixing the copyright missing Travis is complaning about

**Comment by sbrunner** [comment]
Thanks @ochriste

**Pull request merged**